example/src/components/cart-display.js

```diff
@@ -14,6 +14,24 @@ const CartDisplay = () => {
     reduceItemByOne,
   } = useStripeCart();
 
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+    const response = await fetch('/.netlify/functions/create-session', {
+      method: 'post',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(cartDetails),
+    })
+      .then((res) => {
+        return res.json();
+      })
+      .catch((error) => console.log(error));
```

Comment:
At  the moment we are capturing errors to the console - should we throw errors show that users can pick it up in their error handling.

Reply:
I think this is a great idea! Yes

Reply:
Well, since this is an example and not part of the library it might make sense to just pass it along with `console.error` so that the JS doesn't crash but the devs can still see the error. Alongside that, `.catch` should probably be replaced with a `try { await fetch().then(/* toJSON */) } catch { /* handleError */ }`.

Also, should `redirectToCheckout` be called if the `fetch` call to create a session fails?

Reply:
@ChrisBrownie55 if it fails, it should not be called, good catch!